src/content/docs/dev/send-tokens/sui/intro.mdx

```diff
@@ -47,19 +85,70 @@ Once triggered, ITS will generate a unique [token id](#token-id) from the coin's
 
 See [here](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/sui/its-example.js#L195) for an example of registering a new coin.
 
-### Deploy Remote Interchain Token
+## Register Coin Metadata
+
+The `register_coin_from_metadata()` function is used to register the [TokenMetadata](https://github.com/axelarnetwork/axelar-amplifier/blob/main/packages/interchain-token-service-std/src/primitives.rs#L82) msg type with [ITS Hub](/dev/amplifier/its-hub/introduction/) in order to register token data in ITS hub.. This function is required to be called on each chain where you token is being integrated to ITS on if you are planning on linking a custom token. 
+
```

Comment:
> ~~The `register_coin_from_metadata()` function is used to register the [TokenMetadata](https://github.com/axelarnetwork/axelar-amplifier/blob/main/packages/interchain-token-service-std/src/primitives.rs#L82) msg type with [ITS Hub](/dev/amplifier/its-hub/introduction/) in order to register token data in ITS hub..~~

The `register_coin_from_metadata()` function is used to register the [TokenMetadata](https://github.com/axelarnetwork/axelar-amplifier/blob/main/packages/interchain-token-service-std/src/primitives.rs#L82) msg type with [ITS Hub](/dev/amplifier/its-hub/introduction/) in order to register token data in ITS hub.

Reply:
dont see any difference here, can you elaborate?